app-emulation/retroarch/retroarch-1.7.1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic
+
+DESCRIPTION="A sophisticated libretro frontend for emulators, game engines and media players"
+HOMEPAGE="https://retroarch.com/"
+SRC_URI="https://github.com/libretro/RetroArch/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+IUSE="alsa cg +opengl +pulseaudio sdl X"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+LICENSE="GPL-3"
+SLOT="0"
+
+DEPEND="
+	dev-libs/libxml2
+	>=media-libs/freetype-2.8
+	alsa? ( media-sound/alsa-utils )
+	opengl? ( virtual/opengl )
+	pulseaudio? ( media-sound/pulseaudio )
+	sdl? ( media-libs/libsdl2 )
+	X? ( x11-libs/libX11 x11-apps/xinput )"
+
+RDEPEND="
+	${DEPEND}
+	cg? ( media-gfx/nvidia-cg-toolkit )"
+
+S="${WORKDIR}/RetroArch-${PV}"
+
+src_prepare() {
+	append-cflags "-v"
+	default
+}
+
+src_configure() {
+	./configure || die
```

Comment:
You should use `econf` instead of `./configure` [1]. In this case, since you don't need to pass any arguments to `econf`, you can just omit `src_configure` entirely and use the default [2].

[1] https://devmanual.gentoo.org/ebuild-writing/functions/src_configure/configuring/index.html
[2] https://devmanual.gentoo.org/ebuild-writing/functions/src_configure/index.html

Reply:
Actually, since this package has optional dependencies, you will probably need to enable/disable them with `econf` arguments.